rule-engine/rule-engine-components/src/main/java/org/thingsboard/rule/engine/kafka/TbKafkaNode.java

```diff
@@ -91,7 +92,14 @@ public void init(TbContext ctx, TbNodeConfiguration configuration) throws TbNode
         properties.put(ProducerConfig.LINGER_MS_CONFIG, config.getLinger());
         properties.put(ProducerConfig.BUFFER_MEMORY_CONFIG, config.getBufferMemory());
         if (config.getOtherProperties() != null) {
-            config.getOtherProperties().forEach(properties::put);
+            config.getOtherProperties().forEach((k, v) -> {
+                if (SslConfigs.SSL_KEYSTORE_CERTIFICATE_CHAIN_CONFIG.equals(k)
+                        || SslConfigs.SSL_KEYSTORE_KEY_CONFIG.equals(k)
+                        || SslConfigs.SSL_TRUSTSTORE_CERTIFICATES_CONFIG.equals(k)) {
+                    v = v.replace("\\n", "\n");
```

Comment:
I don't like this approach. How about using regexp that also supports other special characters? 
Please add a unit test to make sure everything works as expected.